cli.js

```diff
@@ -39,12 +41,23 @@ function render() {
 		return;
 	}
 
-	logUpdate([
+	var output = [
 		'',
 		'      Ping ' + getSpinner('ping') + ' ' + stats.ping,
 		'  Download ' + getSpinner('download') + ' ' + stats.download,
 		'    Upload ' + getSpinner('upload') + ' ' + stats.upload
-	].join('\n'));
+	];
+
+	if (cli.flags.verbose && stats.data) {
+		output = output.concat([
+			'',
+			'    Server   ' + chalk.cyan(stats.data.server.host),
+			'  Location   ' + chalk.cyan(stats.data.server.location + chalk.dim(' (' + stats.data.server.country + ')')),
+			'  Distance   ' + chalk.cyan(roundTo(stats.data.server.distance, 1) + chalk.dim(' km'))
+		]);
```

Comment:
I was thinking 2, but looking at your example now, I think 1 would be more consistent.